### ISSUE TITLE
Keep Screen On settings affect device screen settings

### ISSUE THREAD

**Issue by KindlyOnes** [koreader/koreader#12694]
* KOReader version:v2024.07-271-g8b710d183_2024-10-30
* Device:
Android: 21
samsung
Model: GT-I9500
ROM: LRX22C.I9500XXUHPJ3

#### Issue
When I set the screen to not turn off while reading in Koreader, the device's screen is set to the maximum time of 10 minutes.
There is no such problem with other readers installed on the device. And there was no such problem in the Korean version a few versions ago. But I don't remember which version it was.

#### Steps to reproduce
Set koreader settings to not turn off the screen.
Go to device settings 
The device settings are set to ten minutes.

**Comment by KindlyOnes** [comment]
The device's screen light duration only changes when the coreader is properly closed. That is, pressing the back button several times and closing the koreader with the exit button. 
It even causes the app to crash. The log record is below.

If I exit the koreader using the home button and delete it from the recently used apps, the device's settings do not change.

[Log_2024-11-03_19-30-29.txt](https://github.com/user-attachments/files/17611279/Log_2024-11-03_19-30-29.txt)

**Comment by pazos** [comment]
Hi, the ticket is invalid.

The sympthoms you have are compatible with custom timeouts but no with "keep screen on".
Custom timeouts indeed depend on the app lifecycle and will fail to restore system timeout if the app crashes.

It seems your device contains a broken rom (or, to be extra fair a rom that triggers a bug that cannot be reproduced on regular android/aosp). That cannot be fixed. "Both "keep screen on" and "system defaults" should work fine.

**Comment by KindlyOnes** [comment]
> Hi, the ticket is invalid.
> 
> The sympthoms you have are compatible with custom timeouts but no with "keep screen on". Custom timeouts indeed depend on the app lifecycle and will fail to restore system timeout if the app crashes.
> 
> It seems your device contains a broken rom (or, to be extra fair a rom that triggers a bug that cannot be reproduced on regular android/aosp). That cannot be fixed. "Both "keep screen on" and "system defaults" should work fine.

To be honest, my English is not good at all. I'm sorry.

I can send a video if you want.  The screen dimming time of the device is 30 seconds.  I made the necessary adjustments to prevent the screen from turning off while reading in Koreader Settings.  When I close Koreader using exit, the 30-second screen dimming time of the device increases to 10 minutes, which is the maximum time for the device.

Apart from this, I don't know why my device settings do not change when I close Koreader without using exit, it continues to remain for 30 seconds.  Another installed reading app does not change the device settings.

The device is old, it was S4. What I said is valid for this device. A slightly more modern device does not have this problem.  Unfortunately the S4's screen size is much better for reading books  I just discovered Koreader app and very liked it.  Great especially for pdf files.

Thanks
